solutions/swb-reference/integration-tests/tests/isolated/projects/create.test.ts

```diff
@@ -123,13 +123,13 @@ describe('Create Project negative tests', () => {
         createRequest.costCenterId = 'cc-invalid-cost-center';
       });
 
-      test('it throws 404 error', async () => {
```

Comment:
Not sure how this ever succeeded if we always threw Boom.badRequest and not Boom.notFound. The `error` below needs to be updated from `Not Found` to `Bad Request`. Is this test working?